src/events/message/commands/help/introduce.ts

```diff
@@ -0,0 +1,84 @@
+import { Message, CommandArg } from "../types";
+import { Role, TextChannel } from "discord.js";
+
+const isModerator = (role: Role) => role.name === "admin" || role.name === "mods";
+
+// introduce
+export default async (message: Message, args: CommandArg[]) => {
+  // Authorization
+  const author = message.guild?.members.cache.get(message.author.id);
+  if (!author) return;
+  const isPrivileged = author.roles.cache.some(isModerator);
+  if (!isPrivileged) return;
+
+  // Input validation
+  if (args.length < 1 || args.length > 2) return;
+  const mention = args[0];
+  if (typeof mention !== "object") return;
+  if (mention.type !== "user") return;
+  const user = message.client.users.cache.get(mention.id);
+  if (!user) return;
+  let channelMention, channel;
+  if (args.length > 1) {
+    channelMention = args[1];
+    if (typeof channelMention !== "object") return;
+    if (channelMention.type !== "channel") return;
+    channel = message.client.channels.cache.get(channelMention.id);
+    if (!(channel instanceof TextChannel)) return;
+  }
+
+  // Action
+  const reply = !channel
+    ? INTRO
+    : channel.name === "help-solve"
+    ? HELP_SOLVE
+    : channel.name === "anything"
+    ? ANYTHING
+    : INTRO;
+  try {
+    const dm = await user.createDM();
+    await dm.send(reply);
+  } catch (err) {
+    console.warn(`failed to DM ${user.tag}: ${err.message || "unknown error"}`);
+    await message.channel.send(`<@${mention.id}> ${reply}`);
+  }
+};
+
+const INTRO = `Welcome to the Codewars Discord server!
```

Comment:
I don't think we need this. We have `#readme` and users are shown these when they join too.